src/ServicePulse.Host/vue/package.json

```diff
@@ -26,16 +26,24 @@
   },
   "devDependencies": {
     "@rushstack/eslint-patch": "^1.7.0",
-    "@vitejs/plugin-vue": "^3.2.0",
+    "@tsconfig/node18": "^18.2.2",
+    "@types/node": "^18.19.3",
+    "@typescript-eslint/eslint-plugin": "^6.19.0",
+    "@typescript-eslint/parser": "^6.19.0",
+    "@vitejs/plugin-vue": "^5.0.3",
     "@vue/eslint-config-prettier": "^9.0.0",
+    "@vue/eslint-config-typescript": "^12.0.0",
     "@vue/test-utils": "^2.1.0",
+    "@vue/tsconfig": "^0.5.0",
     "eslint": "^8.56.0",
     "eslint-plugin-github": "^4.10.1",
     "eslint-plugin-vue": "^9.20.1",
     "jsdom": "^20.0.1",
-    "npm-run-all": "^4.1.5",
     "prettier": "^3.2.3",
-    "vite": "^3.2.8",
-    "vitest": "^0.24.3"
+    "typescript": "^5.3.3",
+    "vite": "^5.0.10",
```

Comment:
can we not increase to 5 yet? I made it 5 on a branch earlier and it had some runtime differences which i didn't want to investigate yet. The monitoring branch has settled on 4

Reply:
I did a few manual tests and it seems to be ok, what were the concerns?

Reply:
All warnings were addressed